src/components/yarn.lock

```diff
@@ -1,4 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file should never have existed. We missed it in this PR: https://github.com/Neufund/generic-ico/pull/27/files#diff-053e2c70f5644e6dbba259f10b013aa4